tag-version/action.yml

```
@@ -52,7 +52,7 @@ runs:
       run: |
         if [ ${{ inputs.push_tag }} == "true" ]; then
           git push origin --tags
```

Not related, so feel free to ignore for the time being. We can also push a single tag here:
```suggestion
          git push origin tag $TAG
```

Even though we don't expect there to be additional tags present, this nonetheless ensures that we only push the tag we expected to create, and that it is in fact a tag (as `tag <tag>` resolves to `refs/tags/<tag>`).